### ISSUE TITLE
Laravel 8 Support

### ISSUE THREAD

**Issue by innovazania** [crscheid/php-article-extractor#32]
Gives error in Laravel 8

**Comment by crscheid** [comment]
Hi there, thanks. Can you post the error you receive in Laravel 8?

**Comment by crscheid** [comment]
Closing issue, no information given.

**Comment by mable-au** [comment]
I'm running Laravel 9 with PHP 8.1 and getting the following when running `composer require crscheid/php-article-extractor`

```
  Problem 1
    - crscheid/php-article-extractor 2.5 requires andreskrey/readability.php ^2.1.0 -> satisfiable by andreskrey/readability.php[v2.1.0].
    - crscheid/php-article-extractor 2.5.1 requires fivefilters/readability.php 2.1.0 -> satisfiable by fivefilters/readability.php[v2.1.0].
    - andreskrey/readability.php v2.1.0 requires psr/log ^1.0 -> found psr/log[1.0.0, ..., 1.1.4] but the package is fixed to 3.0.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - fivefilters/readability.php v2.1.0 requires psr/log ^1.0 -> found psr/log[1.0.0, ..., 1.1.4] but the package is fixed to 3.0.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - Root composer.json requires crscheid/php-article-extractor ^2.5 -> satisfiable by crscheid/php-article-extractor[2.5, 2.5.1].
```

**Comment by ericbae** [comment]
FYI, I'm running Laravel 9 with PHP v8.1 and getting the following error when running `composer require crscheid/php-article-extractor`

```
  Problem 1
    - crscheid/php-article-extractor 2.5 requires andreskrey/readability.php ^2.1.0 -> satisfiable by andreskrey/readability.php[v2.1.0].
    - crscheid/php-article-extractor 2.5.1 requires fivefilters/readability.php 2.1.0 -> satisfiable by fivefilters/readability.php[v2.1.0].
    - andreskrey/readability.php v2.1.0 requires psr/log ^1.0 -> found psr/log[1.0.0, ..., 1.1.4] but the package is fixed to 3.0.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - fivefilters/readability.php v2.1.0 requires psr/log ^1.0 -> found psr/log[1.0.0, ..., 1.1.4] but the package is fixed to 3.0.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - Root composer.json requires crscheid/php-article-extractor ^2.5 -> satisfiable by crscheid/php-article-extractor[2.5, 2.5.1].
```

**Comment by crscheid** [comment]
Great thanks. I’ll take a look. I’m not on Laravel 9 yet so this will be helpful. 👍🏻
